tiledb/sys/build/repo.rs

```diff
@@ -0,0 +1,28 @@
+use std::process::{Command, Stdio};
+
+use crate::error::{Error, Result};
+use crate::utils;
+
+/// Clone TileDB-Inc/TileDB into `target/repos/tiledb`.
+pub fn update() -> Result<()> {
+    if utils::git_dir().is_dir() {
+        return Ok(());
+    }
+
+    let out_dir = utils::out_dir().display().to_string();
+    let output = Command::new("git")
+        .arg("clone")
+        .arg("https://github.com/TileDB-Inc/TileDB")
```

Comment:
Maybe this should be environmental so a network connection isn't required.  So that you can point at a local filesystem path.  That's also potentially useful for testing against non-master.

Similarly, should there be an argument to clone the latest release? or nightly? or something?

Reply:
Yep, I've not added this as I'm not yet certain how we'll end up specifying versions of tiledb to link against when we get to formalized releases of tiledb-rs. I've opened a story here: https://app.shortcut.com/tiledb-inc/story/59243